docs/src/main/sphinx/release/release-423.md

```diff
@@ -86,7 +86,6 @@
 
 ## Hudi connector
 
-* Improve memory accounting of the Parquet writer. ({issue}`18564`)
```

Comment:
@raunaqmorarka  please correct me if I am wrong

Reply:
ah right, forgot that we don't allow writes in Hudi